examples/bootstrap.php

```diff
@@ -26,4 +26,4 @@
 /**
  * Load the credential for the different services
  */
-require_once __DIR__ . '/init.php';
+require_once __DIR__ . '/init.example.php';
```

Comment:
Both of these changes need to be reverted, they would break all the other examples.